charts/hono/ci/mongodb-based-device-registry-values.yaml

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2020 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+# profile for installing Hono
+# - with MongoDB based device registry
+# - with MongoDB instance
+# - without monitoring infrastructure
+
+deviceRegistryExample:
+  type: mongodb
+
+mongodb:
+  createInstance: true
```

Comment:
IMHO we need to add
```yaml
mongodb:
  persistence:
    enabled: false
```
here in order for the install test to succeed because the chart-testing action does not support PVCs ...

Reply:
Thanks for the info. I have updated it.